tezos/api/src/ffi.rs

```diff
@@ -465,25 +459,17 @@ impl From<CallError> for ApplyBlockError {
     fn from(error: CallError) -> Self {
         match error {
             CallError::FailedToCall {
-                parsed_error_message,
-            } => match parsed_error_message {
-                None => ApplyBlockError::FailedToApplyBlock {
-                    message: "unknown".to_string(),
+                error_id,
+                trace_message,
+            } => match error_id.as_str() {
+                "ffi.unknown_predecessor_context" => ApplyBlockError::UnknownPredecessorContext {
```

Comment:
@tizoc 
I am thinking about maybe some module just with this constants,
placed maybe here: tezos/api/src/ocaml_conv/mod.rs as part of a conversion

mod ffi_errors {
 const XYZ: &'static str = "ffi.unknown_predecessor_context"
}

that will match ffi_errors.ml:
https://gitlab.com/simplestaking/tezos/-/merge_requests/15/diffs#5494639cbe5ba8fc72bde25a5b422986f92c6364_12_48

Reply:
Done.